controllers/operator_controller_test.go

```diff
@@ -80,6 +80,38 @@ var _ = Describe("Operator Controller Test", func() {
 				Expect(cond.Message).To(Equal(fmt.Sprintf("package '%s' not found", pkgName)))
 			})
 		})
+		When("the operator specifies a version range for which there are no bundles", func() {
+			var pkgName string
+			BeforeEach(func() {
+				By("initializing cluster state")
+				pkgName = fmt.Sprintf("non-existent-%s", rand.String(6))
```

Comment:
Should this be a package name that _does_ exist, but one that doesn't have a bundle in this range?

Reply:
Gotcha now. I've addressed it. Changed the name of the package and put a comment to say the version of the package is doesn't exist